src/praisonai/praisonai/sandbox/ssh.py

```
@@ -0,0 +1,502 @@
+            
+        except asyncio.TimeoutError:
+            return SandboxResult(
+                execution_id=execution_id,
+                status=SandboxStatus.TIMEOUT,
+                error="Execution timed out",
+                started_at=started_at,
+                completed_at=time.time(),
+                duration_seconds=time.time() - started_at,
+                metadata={"host": self.host, "language": language}
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)